tools/kliopt/org/jetbrains/kliopt/KliOpt.kt

```diff
@@ -17,6 +17,21 @@
 
 package org.jetbrains.kliopt
 
+enum class DeprecatedState {
+    ALL {
+        override fun helpMessage(descriptor: Descriptor): String = "Warning: Option/argument -${descriptor.longName} is deprecated!"
```

Comment:
It makes sense to mention both short and long variants here, e.g. `"Warning: Option/argument -do/-deprecatedOption is deprecated!"`